moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -503,9 +503,15 @@ bool ServoCalcs::cartesianServoCalcs(geometry_msgs::msg::TwistStamped& cmd,
     return false;
 
   // Transform the command to the MoveGroup planning frame
+  if (cmd.header.frame_id.empty())
+  {
+    RCLCPP_WARN_STREAM_THROTTLE(LOGGER, *node_->get_clock(), ROS_LOG_THROTTLE_PERIOD,
+                                "No frame specified for command ,will use planning_frame");
```

Comment:
```suggestion
                                "No frame specified for command, will use planning_frame");
```

also... can you print in there what the planning frame is? It's just in the next line below.